test/testcafe/framework/shared/index.js

```diff
@@ -1,4 +1,9 @@
-import { t } from 'testcafe';
+import { t, ClientFunction } from 'testcafe';
+
+const rerenderWidget = ClientFunction((settings) => {
```

Comment:
Aaron rename this to `renderWidget` in his PR which seems good enough. (no need prefix `re`)